Automattic/woocommerce-subscriptions-core
# v1.3.0 release.

## Description

This PR:
- Updates changelog release date for `1.3.0`.
- Updates version in `package.json` and `woocommerce-subscriptions-core.php` to `1.3.0`.
- Updates `package-lock.json` as a result of running `npm install`.
- Updates `build/index.asset.php` and `build/index.js` as a result of running `npm run build`.

## How to test this PR

- Make sure release date for `1.3.0` is correct.
- Make sure versions are updated in `package.json` and `woocommerce-subscriptions-core.php`.

## Product impact

WooCommerce Payments and WooCommerce Subscriptions.

- [x] Will this PR affect WooCommerce Subscriptions? not until WooCommerce Subscriptions loads `1.3.0`. 4290-gh-woocommerce/woocommerce-subscriptions
- [x] Will this PR affect WooCommerce Payments? not until WooCommerce Payments loads `1.3.0`. https://github.com/Automattic/woocommerce-payments/pull/3550
